packages/devtools/src/schema/parser.ts

```diff
@@ -140,6 +155,18 @@ export class SchemaParser {
     return this.#def
   }
 
+  _parseIndices(directives: Array<DirectiveAnnotation>): Array<FieldsIndex> {
+    return directives.flatMap((d) => {
+      if (d.name === 'createIndex' && d.args) {
+        let indexName = undefined
+        const fields = d.args.fields as Array<Field>
+        return [{ name: indexName, fields }]
```

Comment:
```suggestion
        return [{ fields: d.args.fields as Array<Field> }]
```
The `name` is always `undefined` so this could simplified a bit